src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -779,4 +780,56 @@ public void testYouCantWriteIntoAnExistingDirectory(){
         final String workspace = createTempDir("workspace").getAbsolutePath();
         writeToGenomicsDB(LOCAL_GVCFS, INTERVAL, workspace, 0, false, 0, 1);
     }
+
+    private void cleanupGCSFolder(String path) {
```

Comment:
Instead of implementing this cleanup method here, could you add a `BucketUtils.deleteRecursively(Path)` method, and use the slightly cleaner approach below:

```
public static void deleteRecursively(final Path rootPath) {
    final List<Path> pathsToDelete = Files.walk(rootPath).sorted(Comparator.reverseOrder()).collect(Collectors.toList());
    for (Path path : pathsToDelete) {
        Files.deleteIfExists(path);
    }
}
```

Then modify `BucketUtils.deleteOnExit()` to call into `deleteRecursively()` instead of `deleteFile()`. The advantage of this approach is that if you switch to calling `BucketUtils.getTempFilePath()` in your test methods below, the returned temp paths will be automatically scheduled for deletion on JVM exit, and you can get rid of the finally blocks and the `cleanupGCSFolder()` calls in your test code.